.github/workflows/vercel-preview-tumeet.yaml

```
@@ -1,74 +1,45 @@
 name: Vercel Tumeet Preview Deployment
-env:
-  VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
-  VERCEL_PROJECT_ID: ${{ secrets.VERCEL_TUMEET_PROJECT_ID }}
-  TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
-  TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)